src/saveDocs.ts

```diff
@@ -58,6 +63,18 @@ async function addMetaData(pkg: RawPkg): Promise<FinalPkg> {
     },
   };
 
+  const hasFewDownloads =
+    final.downloadsLast30Days <= config.alternativeNamesNpmDownloadsThreshold &&
+    final.jsDelivrHits <= config.alternativeNamesJsDelivrHitsThreshold;
+
+  const addPopularAlternativeNames =
+    final.popular ||
+    (!final.isDeprecated && !final.isSecurityHeld && !hasFewDownloads);
+
+  if (addPopularAlternativeNames) {
+    final._searchInternal.popularAlternativeNames = [];
```

Comment:
```suggestion
    final._searchInternal.popularAlternativeNames = final._searchInternal.alternativeNames;
```

isn't this meant to be something like this?

Reply:
Lol yes, need to update the tests too.